### PR TITLE
Add boost_system library and use new filename: iptables-config.

### PR DESCRIPTION
**Pull request by john-gress**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -75,10 +75,10 @@ set(TEST_HELPER_LIBS ${TCMALLOC})
 find_library(BOOST_THREAD boost_thread PATHS /usr/local/probe/lib )
 list(APPEND TEST_HELPER_LIBS ${BOOST_THREAD})
 
-find_library(BOOST_FILESYSTEM boost_thread PATHS /usr/local/probe/lib )
```
Comment (KjellKod) [inline CMakeLists.txt]: ha ha ha .. good catch.

**Comment by KjellKod** [comment]
where's the code change for the unit test that moved 10G files? The change was to decrease the file size significantly. It was needed to make the test work on VMs

**Comment by john-gress** [comment]
@KjellKod - the 10 GB to 1GB change is in NMStats. See NMStats/test/TCMallocStatsTest.cpp.

**Comment by KjellKod** [comment]
👍

**Comment by craig-cogdill** [comment]
👍

**Pull request merged**